taker-frontend/src/App.tsx

```diff
@@ -62,24 +62,30 @@ const BTC_USD_SHORT_EVENT = "btcusd_short_offer";
 const ETH_USD_LONG_EVENT = "ethusd_long_offer";
 const ETH_USD_SHORT_EVENT = "ethusd_short_offer";
 
+export interface Selection {
+    symbol: Symbol;
+    position: Map<Symbol, string>;
+}
+
+const defaultSelection: Selection = {
+    position: new Map().set(Symbol.btcusd, "long").set(Symbol.ethusd, "long"),
+    symbol: Symbol.btcusd,
+};
+export const SelectionContext = createContext(defaultSelection);
```

Comment:
What do you think of giving this a better name? E.g. `SelectedSymbolContext`?

Reply:
It's not just the symbol but also the position 😉.

Reply:
Still, `selection` does not say much :)